src/libostree/ostree-kernel-args.c

```diff
@@ -81,6 +82,206 @@ _ostree_kernel_args_free (OstreeKernelArgs *kargs)
   g_free (kargs);
 }
 
+void
+_ostree_kernel_args_cleanup (void *loc)
```

Comment:
Each new public API will need docstrings like you see in e.g. `ostree-repo.c`.

Reply:
Thank you for reviewing !